jetty-websocket/websocket-javax-tests/src/test/resources/alt-filter-web.xml

```diff
@@ -12,16 +12,12 @@
   </context-param>
 
   <filter>
-    <filter-name>wsuf-test</filter-name>
+    <filter-name>org.eclipse.jetty.websocket.util.server.WebSocketUpgradeFilter</filter-name>
```

Comment:
Could we simplify this to just `WebSocketUpgradeFilter` instead of the fully qualified class name?

Reply:
I like the fact that it is a long and painful name.... helps indicate that it is special.